Architecture.md

```diff
@@ -36,17 +43,39 @@
 
 #### CI/CD
 
-***TODO: [UNIFIED WORK] describe use of Github Actions as means of CI/CD***
+Each repository has a GitHub Actions CI to execute format check, linter and tests at each push. This ensures a good code quality and a good test coverage for our entire project.
+
+Most projects use our own Actions available in a [central repository](https://github.com/paastech-cloud/.github). 
+This ensures that all projects have the same CI/CD pipeline and that the CI/CD pipeline is easily maintainable.
+Moreover, thanks to Rust easy documentation creation process, a simple CI job is able to build the documentation and deploy it on [GitHub Pages](https://pages.github.com/) for every Rust project.
+Furthermore, at a tag push on main, or on a workflow call, a Docker image is built for every project that has this workflow set up.
```

Comment:
J'aime pas trop la tournure du début de la phrase
Do you want to say every tag by 'at a tag' or a specific tag ?